MAUI/DataForm/getting-started.md

```diff
@@ -0,0 +1,447 @@
+				    fonts.AddFont("OpenSans-Semibold.ttf", "OpenSansSemibold");
+			    });
+
+		    return builder.Build();
+	    }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Creating data object
+
+The `SfDataForm` is a data edit control so, create a data object with details to create a data form based on your business requirement.
+
+Here, the data object named **DataModel** created with some properties.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+
+    public class ContactsInfo
+    {
+        public string FirstName { get; set; }
+    
+        public string MiddleName { get; set; }
+    
+        public string LastName { get; set; }
+    
+        public string ContactNumber { get; set; }
+    
+        public string Email { get; set; }
+   
+        public string Address { get; set; }
+
+        public DateTime? BirthDate { get; set; }
+    
+        public string GroupName { get; set; }
+
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+Initialize the data object in view model class in order to bind in 'DataObject' property of 'SfDataForm'.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    public class DataFormViewModel
+    {
+        public DataModel DataModel {get; set;}
+        
+        public DataFormViewModel()
+        {
+            this.DataModel = new DataModel();
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Set data object to data form
+
+By default, data form auto generate the editors based on the primitive data type in the 'DataObject' property. Please refer the following code to set the 'DataObject' property.
+
+{% tabs %}
+{% highlight MainPage.xaml %}
+
+    <?xml version="1.0" encoding="utf-8" ?>
+    <ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+                xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+                x:Class="GettingStarted.MainPage">
+
+                    <ContentPage.BindingContext>
+                        <local:DataFormViewModel/>
+                    </ContentPage.BindingContext>    
+	                        <dataForm:SfDataForm x:Name="dataForm" 
+						    DataObject="{Binding DataModel}"/>    
+    </ContentPage>
+
+{% endhighlight %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.DataObject = new DataModel();
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+## Defining editors
+
+The data form control automatically generates `DataFormItems` (which has UI settings of data field) based on the data type in the `SfDataForm.DataObject` property. The `DataFormItem` encapsulates the layout of label and editor setting for the data field appearing in the dataform. 
+
+The type of input editor generated for the data field depends on the type and attribute settings of the property. The following table lists the `DataFormItem` and its constraints for generation:
+
+<table>
+<tr>
+<th>Generated DataFormItem Type</th>
```

Comment:
include Editor and Input control loaded with this table.

So the table should contain
Generated DataFormItem Type | Data Type / Attribute | Editor | Input control loaded

Use details from below added table **The following list of editors are supported:** and remove that table

Reply:
updated